test/forge/GuardianTest.sol

```diff
@@ -98,4 +200,40 @@ contract GuardianTest is IntegrationTest {
         assertEq(pendingGuardian, address(0), "pendingGuardian");
         assertEq(submittedAt, 0, "submittedAt");
     }
+
+    function testOwnerRevokeFeeIncreased(uint256 fee, uint256 elapsed) public {
+        fee = bound(fee, FEE + 1, ConstantsLib.MAX_FEE);
+        elapsed = bound(elapsed, 0, TIMELOCK - 1);
+
+        vm.prank(OWNER);
+        vault.submitFee(fee);
+
+        vm.warp(block.timestamp + elapsed);
+
+        vm.expectEmit();
+        emit EventsLib.RevokeFee(OWNER, IPending(address(vault)).pendingFee());
+        vm.prank(OWNER);
+        vault.revokeFee();
+
+        uint256 newFee = vault.fee();
+        (uint256 pendingFee, uint64 submittedAt) = vault.pendingFee();
+
+        assertEq(newFee, FEE, "newFee");
+        assertEq(pendingFee, 0, "pendingFee");
+        assertEq(submittedAt, 0, "submittedAt");
+    }
+
+    function testGuardianRevokeFeeIncreased(uint256 fee, uint256 elapsed) public {
```

Comment:
So I reverted the suggestion (which was "testOwnerShouldNotRevokeFeeIncreased") optimistically but I'm still open to discuss: this seems like our past convention but we no longer use "should" and "should not" ; there are many counter-examples where we expect to revert but just mention the error's name or simply describe the test case without stating whether we expect to revert or not
In fact, it can avoid to change test names when the feature changes